tools/ci/osquery-ubuntu18.04-toolchain.dockerfile

```diff
@@ -1,36 +1,78 @@
-FROM ubuntu:18.04
-COPY *.deb ./
-RUN apt update -q -y && apt upgrade -q -y && apt install -q -y --no-install-recommends \
-  git \
-  make \
-  cppcheck \
-  ccache \
-  python \
-  python3 \
-  sudo \
-  wget \
-  ca-certificates \
-  tar \
-  icu-devtools \
-  flex \
-  bison \
-  xz-utils \
-  python3-setuptools \
-  python3-psutil \
-  python3-pip \
-  python3-six \
-  rpm \
-  dpkg-dev \
-  file \
-  elfutils \
-  locales \
-  python3-wheel \
-&& dpkg -i linux-base_1.0_all.deb linux-firmware_1.0_all.deb linux-generic_1.0_all.deb \
```

Comment:
Note that I'm dropping this. @Smjert seemed to think we might not need it. If we do, we can bring them back

Reply:
That https://github.com/osquery/osquery/pull/7012 passed implies we don't need this